interface-definitions/snmp.xml.in

```diff
@@ -79,6 +79,39 @@
             </properties>
           </leafNode>
           #include <include/generic-description.xml.i>
+          <node name="mib">
+            <properties>
+              <help>Management information base (MIB)</help>
+            </properties>
+            <children>
+              <leafNode name="interface-max">
+                <properties>
+                  <help>Sets the maximum number of interfaces included in IF-MIB data collection</help>
+                  <valueHelp>
+                    <format>u32:1-4294967295</format>
+                    <description>Sets the maximum number of interfaces included in IF-MIB data collection</description>
+                  </valueHelp>
+                  <constraint>
+                    <validator name="numeric" argument="--range 1-4294967295"/>
+                  </constraint>
+                </properties>
+              </leafNode>
+              <leafNode name="interface">
+                <properties>
+                  <help>Sets the interface name prefix to include in the IF-MIB data collection</help>
+                  <valueHelp>
+                    <format>txt</format>
```

Comment:
Should we probably include proper `<valueHelp>` nodes for the interface types?

Like eth,br,bond.wireguard,vxlan and so on? So the users know only the text part is required and nothing like an asterisk as glob pattern?

Reply:
@c-po Done!